CMakeLists.txt

```diff
@@ -59,7 +59,7 @@ set(BLT_CXX_STD "c++11" CACHE STRING "Version of C++ standard")
 if (ENABLE_CUDA)
   cmake_minimum_required(VERSION 3.9)
 else ()
-  cmake_minimum_required(VERSION 3.8.2)
+  cmake_minimum_required(VERSION 3.8)
```

Comment:
This is a mistake, I'm going to fix it prior to merge.

Reply:
lol - I wondered about that but thought maybe it was intentional